test/elements/test_folder.py

```diff
@@ -108,8 +108,11 @@ def test_update_folder(self, m):
                 "version": 1,
             },
         )
-        folder = self.grafana.folder.update_folder(title="Departmenet DEF", uid="nErXDvCkzz", version=1, overwrite=True)
+        folder = self.grafana.folder.update_folder(
+            title="Departmenet DEF", uid="nErXDvCkzz", new_uid="nErXDvCkzz", version=1, overwrite=True
+        )
         self.assertEqual(folder["title"], "Departmenet DEF")
+        self.assertEqual(folder["uid"], "nErXDvCkzz")
```

Comment:
Wouldn't this work already?
```suggestion
        self.assertEqual(folder["uid"], "oFsYEwDlaa")
```